src/main/java/com/uwetrottmann/tmdb2/services/CollectionService.java

```diff
@@ -22,28 +25,62 @@
 import retrofit2.http.GET;
 import retrofit2.http.Path;
 import retrofit2.http.Query;
+import retrofit2.http.QueryMap;
+
+import java.util.Map;
 
 public interface CollectionService {
     /**
      * Get the basic collection information for a specific collection id.
      *
-     * @param tmdbId TMDb id.
-     * @param language <em>Optional.</em> ISO 639-1 code.
+     * @param collectionId TMDb id.
```

Comment:
I'm not sure i folllow you, what should be the existing Javadoc style? I was seeing multiple  different names for the id on your code. Please enlighten me